analysis_runner/cli_analysisrunner.py

```diff
@@ -35,6 +35,19 @@ def add_analysis_runner_args(parser=None) -> argparse.ArgumentParser:
 
     add_general_args(parser)
 
+    parser.add_argument(
+        '--container',
+        help=(
+            'Container name, if using standard / full access levels, this must start with '
+            'australia-southeast1-docker.pkg.dev/cpg-common/*.'
+        ),
+    )
+    parser.add_argument('--cpu', type=int, help='Number of CPUs to request')
```

Comment:
Might also be worth mentioning the Hail Batch docs here, as that explains how to request mCPU, for example. Does that work with Cromwell too, btw?

Reply:
Yep, I've allowed you to specify it like hail batch now and added links to docs.
I haven't enabled this for Cromwell, because all it does is prepare and deploy, no processing is done, so nothing should really be able to be configured.